### ISSUE TITLE
[1.10.2] Portal 2 incompatibility with optifine. Fixed ?

### ISSUE THREAD

**Issue by Aang23** [iChun/IssueTracker-Mods#102]
I just get a crash with optifine with the SeeTroughPortals enabled. I am on Linux Ubuntu, last version. GTX 1050 with the latest driver, in singleplayer and multiplayer. Without the option to allow to see trough, no problem. Without optifine and enabled, no problem. But as I always use shaders it is hard to stop using them... Before disabling the optin to see trough portals, I want to be sure it's impossible to make this work.
And yes I know it has already been reported, but this is really a major issue, and note that's some other remote rendering mods ( lookinglass for example ), works without troubles.
However, great work !

https://pastebin.com/eZeMNP98   : The crash

**Comment by BernardoCBranco** [comment]
In the latest PG version the game doesn't crash with optifine installed and "SeeTroughPortals" enabled but you can't see trough the portals. Not sure if it has been fixed for 1.10.2.

Optifine: 1.12.2_HD_U_C6
Portal Gun: 1.12.2-7.0.1

**Comment by BernardoCBranco** [comment]
Just tested with 1.10.2 and it's still crashing, altough the crash seems to be different. 
https://pastebin.com/GkqC4kfK

Forge: 12.18.3.2488
Optifine: 1.10.2_HD_U_E3
PortalGun: 6.0.1-fix-1
IchunUtil: 6.5.0

**Comment by iChun** [comment]
Yeah I didn't change any code in 1.10.2. Don't know what the changes were
in 1.12.2 though but at least it isn't crashing anymore?

**Comment by BernardoCBranco** [comment]
No, altough it does have some issues (e.g. it's not possible to see trough portals), it's not crashing.

**Comment by iChun** [comment]
I'm thinking sp614x might have just left the field in this time round to circumvent this crash. It was after all breaking vanilla/forge  compatibility so it was on them to fix.

**Comment by jtnix** [comment]
Please fix for 1.10.2 if possible?  Thanks!

**Comment by iChun** [comment]
1.10.2 will not be getting any more updates, sorry. This issue has been
addressed for 1.12.2.
